src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.querySelectorAll('tr')];
+
+for (const row of rows) {
+  const newFootTh = document.createElement('th');
+  const tfoot = document.querySelector('tfoot');
+  const newTh = document.createElement('th');
+  const thead = document.querySelector('thead');
+  const newTd = document.createElement('td');
+  const secondCell = row.cells[1];
+  const lastCell = row.lastElementChild;
+
+  if (row.parentElement === thead) {
```

Comment:
The condition checks if the row's parent is `thead`. Ensure that the table structure always includes a `thead` to avoid potential errors.